### DISCUSSION TITLE
support for 2fa via sms (can be sent from twillio to a webook)

### DISCUSSION THREAD

**Discussion by priorwave** [spider-rs/spider#238] [Q&A]
Hey guys, i want to access a private site that sometimes requires sms validation. How would one handle this in spider-rs, is there a listener i can send the sms validation code to ?

**Comment by j-mendez** [comment]
Hi, the webhook side the code is not sent to use in that manner. 

Try using the chrome setting with automation to trigger the input and use wait_for events until you input the SMS code.
